testsuite/features/core/srv_disable_local_repos_off.feature

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Turn "disable_local_repos" feature on highstate off
```

Comment:
I'm not convinced about this tweak...
Customers will not have that tweak...


Are we sure this is the best way to setup?

Reply:
The testsuite require it. I tried without and it failed completly.
Remember: our testsuite is not using repos comming from the server, but enabling local repos to provide the needed packages during the tests.
If we disable all local repos again and again, the testsuite cannot run.

Reply:
Ok, let's keep this approach for now.
But we need to make sure that this feature works also fine in a production environment without this hack in the test environment, removing "manually" one of the salt states of a regular onboarding.

Reply:
Fully agree with @srbarrios . We need to test as customers.

Furthermore, not doing real reposyncs is just the root of so many problems.